src/CplexSolverInterface.jl

```diff
@@ -169,6 +169,7 @@ getconstrsolution(m::CplexMathProgModel) = get_constr_solution(m.inner)
 getreducedcosts(m::CplexMathProgModel) = get_reduced_costs(m.inner)
 getconstrduals(m::CplexMathProgModel) = get_constr_duals(m.inner)
 getrawsolver(m::CplexMathProgModel) = m.inner
+getnodecount(m::CplexMathProgModel) = get_node_count(m.inner)
```

Comment:
Not if  ``get_node_count`` keeps the behavior that a zero returned by ``cpxgetnodecnt``should  throw an error.